LDM-732.tex

```diff
@@ -33,12 +33,7 @@
 \setDocDate{\today}
 
 \setDocAbstract {
-The Vera C Rubin Observatory Network Verification Document (VNVD) and associated JIRA V\&V
-Project define the flow-down of specifications from higher level documents to the LSST
-Observatory Network (as defined in LSE-78 LSST Observatory Network), and the methods and
-resources that will be used to verify that the networks have met the specifications satisfactory
-for accepting the Summit Network into DM Subsystem Integration Test (DMSSIT) and LSST
-System Integration Test (SIT)
+The Vera C Rubin Observatory Network Verification Document (VNVD) and associated JIRA V\&V Project define the flow-down of specifications from higher level documents to the LSST Observatory Network (as defined in \citeds{LSE-78} LSST Observatory Network), and the methods and resources that will be used to verify that the networks have met the specifications satisfactory for accepting the Summit Network into DM Subsystem Integration Test (DMSSIT) and LSST System Integration Test (SIT)
```

Comment:
VNVD - is the first V for Vera? - The short version of Vera C Rubin Observatory is just "Rubin" so wou;dn't it be RNVD? (If my intial assumption is correct?) If not, what is VNVD?

Reply:
This is the first time I've seen that acronym.  I suspect that your guess is correct the V is probably for Vera.

I'm also not a fan of using "Document" in the title of a document.  Is it possible to change the title?  Maybe s/document/manfiest/?  Do we need a short handle to refer to this document?  This could be punted on by changing the sentence to  "This document and associated...".

Reply:
OK, so this document contains the verification elements associated with the requirements in LSE-61, How about just: 
"Network Verification Elements" . I don't think we need "Vera C Rubin Observatory" - that is on the doc cover and we dont have it for the DMSR

Reply:
That sounds good to me.